tests/kafkatest/services/performance/end_to_end_latency.py

```diff
@@ -17,9 +17,11 @@
 from kafkatest.services.security.security_config import SecurityConfig
 
 from kafkatest.services.kafka.directory import kafka_dir
+from kafkatest.services.kafka.version import TRUNK, LATEST_0_8_2
 
 
 class EndToEndLatencyService(PerformanceService):
+    MESSAGE_BYTES = 21  # 0.8.X messages are fixed at 21 bytes, so we'll match that for other versions
```

Comment:
It's probably worth double-checking whether this assertion is true

From what I remember, the 0.8.X version of EndToEndLatency sends a hard-coded message, and I computed it at 21 bytes, but it's possible that past Geoff made a mistake